en/docs/references/handle-non-ballerina-files-in-choreo.md

```diff
@@ -0,0 +1,23 @@
+# Handle Non-Ballerina Files in Choreo
+You can add and use non-Ballerina files such as .pem, .txt, etc. in the integration solution you are developing in Choreo. You can add these files to the web editor (Choreo low-code editor) or keep them in your GitHub repository. 
+
+Running and testing this solution with non-Ballerina files in the web editor will give you results as expected as the files are available in the environment. However, at build time, Choreo only copies the `Cloud.toml` and the Ballerina target/jar files to the Docker container. The target/jar does not by default include the non-Ballerina files we add into Choreo. Therefore, **Choreo does not copy the non-Ballerina files to the Docker container unless explicitly mentioned in the `Cloud.toml` file**. This may lead to a **`no such file or directory`** error at runtime.
```

Comment:
Shall we avoid the term 'we'.

Also perhaps rephrase to 
"If you run and test this solution with non-Ballerina files in the web editor, you get the expected outcome  . .  . . "

Reply:
fixed.